File: gabriel_goncalves/tests/unit/CoffeeTest.php
<?php

namespace App\Test;

use \PHPUnit_Framework_TestCase;
use App\Coffee;

class CoffeeTest extends PHPUnit_Framework_TestCase
{
    public function testVerificaSeTemCafeEDeveRetornarFalsoPoisEstaVazio()
    {
        $coffee = new Coffee();
        $coffee->refill();
        $this->assertTrue($coffee->isEmpty());
    }

    public function testDeveRetornarACanecaCheiaPoisFoiPreenchida()
    {
        $coffee = new Coffee();
        $this->assertFalse($coffee->isEmpty());
    }


    public function testDeveRetornarVaziaPorqueOCafeFoiTomado()
    {
        $coffee = new Coffee();
        $coffee->refill();
        $this->assertTrue($coffee->isEmpty());
    }
}

File: matheus_marabesi/test/CoffeeTest.java
import static org.junit.Assert.*;

import org.junit.Test;

public class CoffeeTest {

	@Test
	public void mugShouldBeEmpty() {
		Coffee coffee = new Coffee();
		
		assertTrue("Mug with coffee should be empty", coffee.isEmpty());
	}

	@Test
	public void shouldRefillTheMugWithCoffee() throws Exception{
		Coffee coffee = new Coffee();
		coffee.refill();
		
		assertFalse(coffee.isEmpty());
	}
	
	@Test
	public void shouldDrinkCoffeeWhenItIsAvailable() throws Exception {
			Coffee coffee = new Coffee();
			coffee.refill();
			coffee.drink();
	}
}
